### PR TITLE
feat: add meta.yml for IS2view

### PR DESCRIPTION
**Pull request by tsutterley**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tsutterley** [comment]
I will get back to this after I build a new release. Needed to add a manifest file.

**Comment by tsutterley** [comment]
I think this is ready for review at your convenience 👍

File: recipes/is2view/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{% set name = "IS2view" %}
+{% set version = "0.0.4" %}
+{% set sha256 = "a6d9d02c6bd993ad128e8f5c1dc502c956e36cb81a63e121316cc43a4f23f632" %}
```
Comment (ocefpaf) [inline recipes/is2view/meta.yaml:3]:
```suggestion
```

File: recipes/is2view/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - setuptools
+  run:
+    - python >=3.6
+    - ipyleaflet
+    - matplotlib-base
+    - netcdf4
+    - numpy
+    - rioxarray
+    - setuptools-scm
+    - setuptools
```
Comment (ocefpaf) [inline recipes/is2view/meta.yaml:34]: These are probably a mistake upstream, I don't think `setuptools-scm` and `setuptools` are needed at run time. 

```suggestion
```

**Pull request merged**